test_dev.sh

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env bash
+
+# Catch any non-flagged use
+if [ $# -eq 0 ]; then
+    echo "Invalid: $(basename "$0") has required flags, use: -h for help"
+    exit 1
+fi
+
+# e: Stops execution in the instance of a command or pipeline error
+# u: Treat unset variables as an error and exit immediately
+set -eu
+
+# Default to FAILURE
+STATUS="FAILURE"
+
+function outcome_and_exit {
+  if [ $STATUS == "SUCCESS" ]
+  then
+    echo -e "\e[32m$STATUS\e[0m"
+  else
+    echo -e "\e[31m$STATUS\e[0m"
+  fi
+}
+
+while getopts ":ha" opt; do # Parse options to the `test_cmd` command
+  case ${opt} in
+    h)
+      echo
+      echo "Usage:"
+      echo "    -h             Displays this help message"
+      echo "    -a             Run all unit and integration tests"
```

Comment:
nit: this also runs lints:
https://github.com/letsencrypt/boulder/blob/cd3a06b76c4edd0b5e84b361cc85a194e57bffab/test.sh#L17